pydrive2/test/test_apiattr.py

```diff
@@ -2,20 +2,27 @@
 
 from pydrive2.auth import GoogleAuth
 from pydrive2.drive import GoogleDrive
+from pydrive2.test.test_util import pydrive_retry, setup_credentials
 
 class ApiAttributeTest(unittest.TestCase):
     """Test ApiAttr functions.
     """
-    ga = GoogleAuth('settings/test1.yaml')
-    ga.LocalWebserverAuth()
-    first_file = 'a.png'
-    second_file = 'b.png'
+
+    first_file = 'pydrive2/test/files/a.png'
+    second_file = 'pydrive2/test/files/b.png'
+
+    @classmethod
+    def setup_class(cls):
+        setup_credentials()
+
+        cls.ga = GoogleAuth('pydrive2/test/settings/test1.yaml')
```

Comment:
why do we need to share this on the class level?

Reply:
I've moved it into the `setUp` method, since `drive` instance requires it and is supposed to be used by all test methods.

In other TestCase classes `drive` instance usually is not shared across all test methods and separate instance is created per each test method. Not sure if it makes any difference with sharing of single `drive` instance.